compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/test/canonicalize.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: iree-opt --split-input-file --canonicalize %s | FileCheck %s
+
+//===----------------------------------------------------------------------===//
+// amdiae.LogicalObjectFifoFromMemref
+//===----------------------------------------------------------------------===//
+
+// CHECK-LABEL: func.func @logicalobjectfifo_from_memref
+// CHECK:       %[[C0:.+]] = arith.constant 0 : index
+// CHECK:       %[[C1:.+]] = arith.constant 1 : index
+// CHECK:       %[[TILE_0_0:.*]] = amdaie.tile(%[[C0]], %[[C0]])
+// CHECK:       %[[TILE_0_1:.*]] = amdaie.tile(%[[C0]], %[[C1]])
+// CHECK:       %[[TILE_1_0:.*]] = amdaie.tile(%[[C1]], %[[C0]])
+// CHECK:       %[[TILE_1_1:.*]] = amdaie.tile(%[[C1]], %[[C1]])
+// CHECK:       %[[I0:.*]] = amdaie.logicalobjectfifo.from_memref %[[ARG0:.*]], {%[[TILE_0_0]], %[[TILE_0_1]], %[[TILE_1_0]], %[[TILE_1_1]]}
```

Comment:
No need to match `%[[I0:.*]]`.

Just do `amdaie.logicalobjectfifo.from_memref %[[ARG0:.*]], {%[[TILE_0_0]], %[[TILE_0_1]], %[[TILE_1_0]], %[[TILE_1_1]]}`

Reply:
Done.